src/Digdir.Domain.Dialogporten.WebApi/Endpoints/V1/ServiceOwner/Dialog/DeleteDialogEndpoint.cs

```diff
@@ -38,6 +46,27 @@ public sealed class DeleteDialogRequest
 {
     public Guid DialogId { get; set; }
 
-    [FromHeader(headerName: Constants.IfMatch, isRequired: false)]
+    [FromHeader(headerName: Constants.IfMatch, isRequired: false, removeFromSchema: true)]
     public Guid? ETag { get; set; }
-}
\ No newline at end of file
+}
+
+public sealed class DeleteDialogEndpointSummary : Summary<DeleteDialogEndpoint>
+{
+    public DeleteDialogEndpointSummary()
+    {
+        Summary = "Deletes a dialog";
+        Description = """
+                Deletes a given dialog (soft delete). For more information see the documentation (link TBD).
+
+                Note that the dialog will still be available on the single details endpoint, but will have a deleted status. It will not appear on the list endpoint for either service owners or end users.
+                If end users attempt to access the dialog via the details endpoint, they will get a 410 Gone response.
+
+                Optimistic concurrency control is implemented using the If-Match header. Supply the ETag value from the GetDialog endpoint to ensure that the dialog is not deleted by another request in the meantime.
+                """;
+        Responses[StatusCodes.Status204NoContent] = "The dialog was deleted successfully";
+        Responses[StatusCodes.Status401Unauthorized] = Constants.SummaryErrorServiceOwner401;
+        Responses[StatusCodes.Status403Forbidden] = "Unauthorized to delete the supplied dialog (not owned by authenticated organization or has additional scope requirements defined in policy)";
+        Responses[StatusCodes.Status404NotFound] = "The given dialog ID was not found or is already deleted";
+        Responses[StatusCodes.Status412PreconditionFailed] = "The supplied If-Match header did not match the current ETag value for the dialog. The dialog was not deleted.";
```

Comment:
```suggestion
                Deletes a given dialog (soft delete). For more information see the documentation (link TBD).

                Note that the dialog will still be available on the single details endpoint, but will have a deleted status. It will not appear on the list endpoint for either service owners nor end users.
                If end users attempt to access the dialog via the details endpoint, they will get a 410 Gone response.

                Optimistic concurrency control is implemented using the If-Match header. Supply the ETag value from the GetDialog endpoint to ensure that the dialog is not modified/deleted by another request in the meantime.
                """;
        Responses[StatusCodes.Status204NoContent] = "The dialog was deleted successfully.";
        Responses[StatusCodes.Status401Unauthorized] = Constants.SummaryErrorServiceOwner401;
        Responses[StatusCodes.Status403Forbidden] = "Unauthorized to delete the supplied dialog (not owned by authenticated organization or has additional scope requirements defined in policy).";
        Responses[StatusCodes.Status404NotFound] = "The given dialog ID was not found or is already deleted.";
        Responses[StatusCodes.Status412PreconditionFailed] = "The supplied If-Match header did not match the current ETag value for the dialog. The dialog was not deleted.";
```

For en eller annen grunn kommer ikke 404 opp i swaggeren 🤔